odoo/addons/base/views/res_bank_views.xml

```diff
@@ -85,8 +85,8 @@
                     <group>
                         <group>
                             <field name="sequence" invisible="1"/>
+                            <field name="acc_type" invisible="1"/>
```

Comment:
Did you talk this through?

Reply:
@jco-odoo It's in the task, as well as the discussion about it :p It was asked by tsb. It's acutally not so big a deal, as a little v next to the account number now indicates if it's correct or not.